ReactAndroid/src/main/java/com/facebook/react/modules/network/RequestBodyUtil.java

```diff
@@ -58,6 +62,17 @@ public static boolean isGzipEncoding(@Nullable final String encodingType) {
       if (fileContentUri.getScheme().startsWith("http")) {
         return getDownloadFileInputStream(context, fileContentUri);
       }
+
+      if (fileContentUriStr.startsWith("data:")) {
+        byte[] decodedDataUrString = Base64.decode(fileContentUriStr.split(",")[1], Base64.DEFAULT);
+        Bitmap bitMap = BitmapFactory.decodeByteArray(decodedDataUrString, 0, decodedDataUrString.length);
+        ByteArrayOutputStream bytes = new ByteArrayOutputStream();
+        bitMap.compress(Bitmap.CompressFormat.JPEG, 100, bytes);  
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
Is this unused?

Reply:
@cortinico yes that variable is unused and i forgot to remove that before commiting.